pkg/security/admission/admission.go

```diff
@@ -92,38 +91,32 @@ func (c *constraint) Admit(a kadmission.Attributes) error {
 		}
 		matchedConstraints = append(matchedConstraints, saConstraints...)
 	}
-
-	// remove duplicate constraints and sort
 	matchedConstraints = oscc.DeduplicateSecurityContextConstraints(matchedConstraints)
 	sort.Sort(oscc.ByPriority(matchedConstraints))
-
-	providers, errs := oscc.CreateProvidersFromConstraints(a.GetNamespace(), matchedConstraints, c.client)
-	logProviders(pod, providers, errs)
-
-	if len(providers) == 0 {
```

Comment:
differentiating between no providers and no matching provider is also a valuable distinction (for an admin troubleshooting), we may want to retain this